### PR TITLE
[ENG-1530] Display item count alongside labels

### PR DESCRIPTION
**Pull request by Brendonovich**
image

### CODE REVIEW

**Comment by Brendonovich** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #2011** <a href="https://app.graphite.dev/github/pr/spacedriveapp/spacedrive/2011?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/spacedriveapp/spacedrive/2011?utm_source=stack-comment).

**Review by brxken128 — APPROVED** [review]

**Review by brxken128 — APPROVED** [review]

File: interface/app/$libraryId/Explorer/View/GridView/Item/index.tsx [hunk 1]
```diff
@@ -138,3 +139,22 @@ const ItemSize = () => {
...
+					in: [data.item.id]
+				}
+			}
+		}]
+	}])
+
+	if(count.data === undefined) return
+
+	return <div className="truncate rounded-md px-1.5 py-[1px] text-center text-tiny text-ink-dull">
+		{count.data} {count.data === 1 ? "item" : "items"}
```
Comment (utkubakir) [inline interface/app/$libraryId/Explorer/View/GridView/Item/index.tsx:157]: i18n?

Comment (Brendonovich) [reply interface/app/$libraryId/Explorer/View/GridView/Item/index.tsx:157]: am i to generate all the translations or just add the english one?

Comment (utkubakir) [reply interface/app/$libraryId/Explorer/View/GridView/Item/index.tsx:157]: up to you, all languages would be nice ofc

Comment (Brendonovich) [reply interface/app/$libraryId/Explorer/View/GridView/Item/index.tsx:157]: brb learning 8 languages

Comment (utkubakir) [reply interface/app/$libraryId/Explorer/View/GridView/Item/index.tsx:157]: haha all of our translations are chatgpt rn anyway so google translate mb?

**Review by utkubakir — APPROVED** [review]

**Pull request merged**